python/auto/graphics/grapher_mpl.py

```diff
@@ -116,7 +121,7 @@ def __init__(self,parent=None,**kw):
         self.ax2d = self.ax
         self.ax3d = None
         if kw.get("hide"):
-            self.canvas = FigureCanvasAgg(self.ax.get_figure())
+            self.canvas = self.ax.get_figure().canvas
```

Comment:
i think in that case we can also remove the
`from matplotlib.backends.backend_agg import FigureCanvasAgg`
above since it's not used elsewhere.

Reply:
yes, that's correct. See updated PR. 

The other thing to think about is if "demos/cusp" is a good home for the notebook example?